app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -97,4 +97,15 @@ def textual_topology
      :link  => url_for(:controller => 'container_topology', :action => 'show', :id => @ems.id),
      :title => N_("Show topology")}
   end
+
+  def textual_volumes
+    count_of_volumes = @ems.number_of(:persistent_volumes)
+    label = ui_lookup(:tables => "volumes")
+    h     = {:label => label, :image => "container_volume", :value => count_of_volumes}
+    if count_of_volumes > 0 && role_allows(:feature => "persistent_volume_show_list")
+      h[:link]  = url_for(:action => 'show', :id => @ems, :display => 'persistent_volumes')
+      h[:title] = "Show all #{label}"
+    end
+    h
+  end
```

Comment:
@zeari, are you talking about `url_for` here?  I suggest that if that's deprecated, then it probably needs a separate refactoring PR (across the board) to get a single way to handle that.  Sound good?